author/views.py

```diff
@@ -1,3 +1,13 @@
-from django.shortcuts import render
+from django.views import generic
+from .models import Author
 
-# Create your views here.
+class BloggerDetailView(generic.DetailView):
+    model = Author
+    template_name = "author/blogger_detail.html"
+    context_object_name = "author"
+    
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        author = self.get_object()
+        context['auth_posts'] = author.blogs.all().order_by('-created_at')
```

Comment:
Rename `auth_posts` to author posts. I think we need pagination for `auth_posts`, as we can have lots of posts for the author.